src/drivers/distance_sensor/ll40ls/LidarLite.h

```diff
@@ -44,22 +44,24 @@
 #include <lib/drivers/rangefinder/PX4Rangefinder.hpp>
 #include <perf/perf_counter.h>
 
+using namespace time_literals;
+
 /* Device limits */
-#define LL40LS_MIN_DISTANCE (0.05f)
-#define LL40LS_MAX_DISTANCE (25.00f)
+#define LL40LS_MIN_DISTANCE    (0.05f)
+#define LL40LS_MAX_DISTANCE    (25.00f)
 #define LL40LS_MAX_DISTANCE_V2 (35.00f)
 
 
-// normal conversion wait time
-#define LL40LS_CONVERSION_INTERVAL 50*1000UL /* 50ms */
+// Normal conversion wait time.
+#define LL40LS_CONVERSION_INTERVAL 50_ms
 
-// maximum time to wait for a conversion to complete.
-#define LL40LS_CONVERSION_TIMEOUT 100*1000UL /* 100ms */
+// Maximum time to wait for a conversion to complete.
+#define LL40LS_CONVERSION_TIMEOUT 100_ms
```

Comment:
While you're at it you could make these `static constexpr auto`, right?

Reply:
Done! Thanks!